Revise CONTRIBUTING doc (#2577)
fishtown-analytics/dbt: .github/ISSUE_TEMPLATE/feature_request.md

### Who will this benefit?
What kind of use case will this feature be useful for? Please be specific and provide examples, this will help us prioritize properly.

### Are you interested in contributing this feature?

> Let us know if you want to write some code, and what help you want in getting started.

Reviewer: ```suggestion
Let us know if you want to write some code, and how we can help.
```
